src/main/java/com/capitalone/dashboard/collector/DefaultSonar6Client.java

```diff
@@ -154,9 +152,9 @@ private void getProjects(String url, String key, JSONArray jsonArray, int pageNu
     }
 
     @Override
-    public CodeQuality currentCodeQuality(SonarProject project, String metrics) {
+    public CodeQuality currentCodeQuality(SonarProject project) {
         String url = String.format(
-                project.getInstanceUrl() + URL_RESOURCE_DETAILS, project.getProjectId(), metrics);
+                project.getInstanceUrl() + URL_RESOURCE_DETAILS, project.getProjectId(), METRICS);
```

Comment:
can we add a check to see if metrics passed is null or empty and then use the default metrics?

Reply:
we want the flexibility of loading the metrics from the properties as well.

Reply:
Hmm, ok. I think the metrics property would be best as a "global" sonar property (sonar.metrics6=blah,blah) but not instance specific (sonar.metrics[2]=quux,ducks). is that fair? and you're saying if metrics are in properties they override the client METRICS right?

Reply:
ok, I'll have something worked out by tomorrow